apps/terminal.asm

```diff
@@ -147,6 +177,17 @@ main:
 .reset:
     int 19h
 
+.shutdown:
+    mov ax, 0x1000
+    mov ax, ss
+    mov sp, 0xf000
+    mov ax, 0x5307
+    mov bx, 0x0001
+    mov cx, 0x0003
+    int 0x15
+
+    ret  ;if interrupt doesnt work
```

Comment:
this thing here is wrong.

Reply:
I tried it and it worked well

Reply:
should I remove these lines?

186 187 188 and 189